rpcs3/cmake_modules/ConfigureCompiler.cmake

```diff
@@ -29,3 +28,61 @@ elseif(${CMAKE_CXX_COMPILER_ID} MATCHES "Clang")
 		add_compile_options(-stdlib=libc++)
 	endif()
 endif()
+
+if(NOT MSVC)
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-invalid-pch -fexceptions")
+	if(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+		set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-unused-command-line-argument")
+	endif()
+
+	# This hides our LLVM from mesa's LLVM, otherwise we get some unresolvable conflicts.
+	if(NOT APPLE)
+		set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wl,--exclude-libs,ALL")
+	endif()
+
+	if(WIN32)
+		set(CMAKE_RC_COMPILER_INIT windres)
+		enable_language(RC)
+		set(CMAKE_RC_COMPILE_OBJECT "<CMAKE_RC_COMPILER> <FLAGS> -O coff <DEFINES> -i <SOURCE> -o <OBJECT>")
+
+		# Workaround for mingw64 (MSYS2)
+		set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wl,--allow-multiple-definition")
+	endif()
+
+	add_compile_options(-msse -msse2 -mcx16)
+
+	if(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+		# This fixes 'some' of the st11range issues. See issue #2516
+		if(APPLE)
+			set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -Wl,-image_base,0x10000 -Wl,-pagezero_size,0x10000")
```

Comment:
I'm pretty sure it's fine to delete these image-base arguments, the real fix was adding `-no-pie`.

Reply:
Removed

Reply:
This might still be needed. I also removed similar MSVC option /BASE:0x10000 and that caused
`PPUThread.cpp:243` line start throwing an exception because ppu_fallback was more than what fits in u32.